spec/quality_spec.rb

```diff
@@ -40,6 +40,22 @@ def check_for_extra_spaces(filename)
     "#{filename} has spaces on the EOL on lines #{failing_lines.join(", ")}"
   end
 
+  def check_for_expendable_words(filename)
+    failing_line_message = []
+    useless_words = /\b(actually|obviously|just|clearly|basically|really)\b/i
+    second_person_words = /\b(he|she)\b/i
```

Comment:
The pronoun checking might be better off extracted into a separate test and method as it's not exactly the same functional domain as expendable modifier words.